ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryEventListener.java

```diff
@@ -54,6 +54,15 @@ default void onStateChange(
   ) {
   }
 
+  /**
+   * Called when the state of the ksqlDB query metadata changes
```

Comment:
This comment makes it sound like the method is called anytime any part of the ksqlDB query metadata changes, but it's only called when the query is paused or resumed, right? If so, let's update the comment to reflect that.

Reply:
Presently that's when it is called.  Updating the java doc.